src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php

```diff
@@ -274,20 +273,6 @@ public function testEnvWithVariableNode(): void
         $this->assertSame($expected, $container->resolveEnvPlaceholders($ext->getConfig()));
     }
 
-    /**
-     * @group legacy
-     */
-    public function testConfigurationWithoutRootNode(): void
```

Comment:
this cannot happen anymore with Config 5.0, in the #31678 the deps are updated .. as such i assumed we dont need this test anymore.